Docs/beta/conf.py

```diff
@@ -167,7 +176,16 @@ def setup(app):
 # default: ``['localtoc.html', 'relations.html', 'sourcelink.html',
 # 'searchbox.html']``.
 #
-html_sidebars = {'**': ['globaltoc.html', 'localtoc.html', 'searchbox.html'] }
+# furo theme sidebars
+html_sidebars = {
+    "**": [
+        "sidebar/brand.html",
+        "sidebar/scroll-start.html",
+        "sidebar/navigation.html",
+#        "sidebar/search.html",
```

Comment:
Was this commented out intentionally? Seems like you do have the css file for search.html, don't you?